### ISSUE TITLE
How to use resumeAfterTermination when the app has been fully quit - iOS

### ISSUE THREAD

**Issue by caleblindsey** [plaid/plaid-link-ios#26]
### A detailed description of the steps to reproduce the issue

* Start link flow with Plaid OAuth app2app (in this case - Chase)
* User kills our app
* User's banking app routes them back to our app
* Use Plaid's `resumeAfterTermination`

_Can you provide instructions or example code on using this API?_

### What you expected to see?

A continuation of the OAuth flow.

### What you saw instead?

It appears to be some sort of internal TODO message left by developers?

<details><summary>Screenshots that show the state of the UI (if applicable)</summary>

<img width="300" src="https://github.com/plaid/plaid-link-ios/assets/100710578/050f6e25-33a7-4034-a550-6588275baa18">

</details>

### Some Relevant Code

```
            handler.resumeAfterTermination(from: url)
            let modalPresentation: LinkKit.PresentationHandler = { [weak self] linkVC in
                guard let self else { return }
                linkVC.isModalInPresentation = true // prevents swiping to dismiss
                self.present(linkVC, animated: true)
            }
            handler.open(presentUsing: .custom(modalPresentation))
```

---

When submitting an issue for [**Plaid Link iOS**](https://github.com/plaid/link/tree/master/ios) please include the following information:

| Name            | Value |
|-----------------|-------|
| Link `env`      |    prod   |
| LinkKit version |    4.6.5   |
| iOS version     |   17.1    |
| iOS device      |    iPhone 15 Pro   |

#### List of used 3rd party libraries:

* [Plaid SPM](https://github.com/plaid/plaid-link-ios)

**Comment by dtroupe-plaid** [comment]
@caleblindsey thanks for reporting. After some investigation this appears to be a bug on our end. We will release an update to patch this ASAP. I'll follow up here once that release is live.

**Comment by caleblindsey** [comment]
> @caleblindsey thanks for reporting. After some investigation this appears to be a bug on our end. We will release an update to patch this ASAP. I'll follow up here once that release is live.

Awesome! Thank you for the response.

**Comment by dtroupe-plaid** [comment]
@caleblindsey I just released [4.7.0](https://github.com/plaid/plaid-link-ios/releases/tag/4.7.0) which resolves this issue. 

The code you posted above should now work

            handler.resumeAfterTermination(from: url)
            let modalPresentation: LinkKit.PresentationHandler = { [weak self] linkVC in
                guard let self else { return }
                linkVC.isModalInPresentation = true // prevents swiping to dismiss
                self.present(linkVC, animated: true)
            }
            handler.open(presentUsing: .custom(modalPresentation))

If you have any more issues related to `resumeAfterTermination` feel free to comment in this issue, but I am going to close it for now.
